shared/src/packages/orders/libs/types/order-find-all-user-orders-response-dto.type.ts

```diff
@@ -0,0 +1,8 @@
+import { type OrderResponseDto } from './order-response-dto.type.js';
+
+type OrderFindAllUserOrdersResponseDto = {
+  items: OrderResponseDto[];
+  total: number;
+};
```

Comment:
we have already type EntityPagination
so you can reuse it -> `EntityPagination<OrderResponseDto>`